style.css

```diff
@@ -637,145 +637,203 @@ ul {
-}
-
-@media (min-width: 1160px) {
-  .header {
-    padding: 0;
-    width: 90%;
-  }
+  background: $background_color;
+  box-shadow: 0 1px 13px 0 rgba(0, 0, 0, 0.1), 0 1px 5px 0 rgba(0, 0, 0, 0.06);
+  height: 80px;
+  padding-top: 10px;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 10;
 }
 
-.logo img {
-  max-height: 37px;
-  vertical-align: middle;
+.header-area {
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas: "logo search menu";
+  -ms-grid-columns: max-content 1fr max-content;
+  grid-template-columns: max-content 1fr max-content;
+  -ms-grid-rows: auto;
+  grid-template-rows: auto;
+  height: 70px;
+  padding: 0 40px;
 }
 
-.logo span {
-  margin: 0 10px;
-  color: $brand_color;
+.header-area > * {
+  -ms-grid-row-align: center;
+  align-self: center;
 }
 
-.logo a {
-  display: inline-block;
+.logo {
+  -ms-grid-row: 1;
+  -ms-grid-column: 1;
+  color: $text_color;
+  display: -ms-grid;
+  display: grid;
+  grid-gap: 8px;
+  gap: 8px;
+  grid-area: logo;
+  grid-template-areas: "svg site-name";
+  -ms-grid-columns: auto;
+  grid-template-columns: auto;
+  padding: 10px;
+  transform: translateX(-10);
 }
 
-.logo a:hover, .logo a:focus, .logo a:active {
+.logo:hover, .logo:active, .logo:focus {
   text-decoration: none;
 }
 
-.user-nav {
-  display: inline-block;
-  position: absolute;
-  white-space: nowrap;
-}
-
-@media (min-width: 768px) {
-  .user-nav {
-    position: relative;
-  }
+.logo svg {
+  -ms-grid-row: 1;
+  -ms-grid-colum: 1;
+  -ms-grid-row-align: center;
+  align-self: center;
+  grid-area: svg;
+  max-width: 125px;
+  width: 100%;
 }
 
-.user-nav[aria-expanded="true"] {
-  background-color: #fff;
-  box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.15), 0 4px 10px 0 rgba(0, 0, 0, 0.1);
-  border: solid 1px #ddd;
-  right: 0;
-  left: 0;
-  top: 71px;
-  z-index: 1;
+.logo span {
+  -ms-grid-row: 1;
+  -ms-grid-column: 3;
+  -ms-grid-row-align: center;
+  align-self: center;
+  font-family: $heading_font;
+  font-size: 10px;
+  font-style: italic;
+  font-weight: 600;
+  grid-area: site-name;
+  letter-spacing: 1px;
+  line-height: 1;
+  overflow: hidden;
+  padding-left: 9px;
+  position: relative;
+  text-transform: uppercase;
 }
 
-.user-nav[aria-expanded="true"] > a {
+.logo span:before {
+  background: #ccc;
+  content: " ";
   display: block;
-  margin: 20px;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 50%;
+  transform: rotate(20deg) translateY(-50%);
+  width: 1px;
 }
 
-.nav-wrapper a {
-  border: 0;
-  color: $link_color;
-  display: none;
-  font-size: 14px;
-  padding: 0 20px 0 0;
-  width: auto;
+.header-spacer {
+  -ms-grid-row: 1;
+  -ms-grid-column: 2;
+  grid-area: search;
 }
 
-@media (min-width: 768px) {
-  .nav-wrapper a {
-    display: inline-block;
-  }
+.nav-wrapper {
+  -ms-grid-row: 1;
+  -ms-grid-column: 3;
+  grid-area: menu;
+  position: relative;
 }
 
-[dir="rtl"] .nav-wrapper a {
-  padding: 0 0 0 20px;
+.nav-wrapper .nav-item {
```

Comment:
I know the www site uses the expanding button pattern, but it is only used to mitigate a lack of horizontal space. We have ample horizontal space in this header. I think we should just have the buttons expanded by default.